core/ExceptionHandler.js

```diff
@@ -14,6 +17,35 @@ import {XH} from './XH';
  */
 export class ExceptionHandler {
 
+    /**
+     * Property paths within error details JSON to replace with REDACT_TEXT
+     * @type {string[]}
+     */
+    static REDACT_PATHS = [
+        'fetchOptions.headers.Authorization'
+    ];
+
+    /**
+     * Text used to replace values that match REDACT_PATHS
+     * @type {string}
+     */
+    static REDACT_TEXT = '[redacted]';
```

Comment:
thinking this can just be fixed as *****.  No need to allow customization -- we don't elsewhere in the API